Week3/homework/2dogyears.js

```diff
@@ -0,0 +1,19 @@
+// **Exercise 2: Dog years**
+//
+// You know how old your dog is in human years, but what about dog years? Calculate it!
+//
+// 1. Write a function named `calculateDogAge`.
+//
+// - It takes 1 argument: your puppy's age (number).
+// - Calculate your dog's age based on the conversion rate of 1 human year to 7 dog years.
+// - Return a string: "Your doggie is [CALCULATED_VALUE] years old in dog years!"
+
+function calculateDogAge(year){
+  return `Your doggie is ${year * 7} years old in dog years!`;
+}
```

Comment:
Ah. There seems to be a typo here as I think what it means is that 7 human years are equal to 1 dog year. Thus, your logic would eventually have to be reversed from `year * 7` to `year / 7`. But that's fine now. 🙂

Reply:
Don't know. Never seen a dog that lived more than 10 years. So, I just assumed this way. :)